test/ck_tile/grouped_gemm/test_grouped_gemm.cpp

```diff
@@ -8,19 +8,29 @@
 #include "ck_tile/host.hpp"
 #include "test_grouped_gemm_util.hpp"
 
-using F16 = ck_tile::half_t;
-using F32 = float;
-
-using Row = ck_tile::tensor_layout::gemm::RowMajor;
-using Col = ck_tile::tensor_layout::gemm::ColumnMajor;
+using F16   = ck_tile::half_t;
+using F32   = float;
+using Row   = ck_tile::tensor_layout::gemm::RowMajor;
+using Col   = ck_tile::tensor_layout::gemm::ColumnMajor;
+using True  = ck_tile::bool_constant<true>;
+using False = ck_tile::bool_constant<false>;
+using KB1   = ck_tile::number<1>;
+using KB2   = ck_tile::number<2>;
```

Comment:
It doesn't have to be a test parameter. You can create a test case which loops over some set of kbatch values and test them - but then bear in mind that not all kbatch values might be supported.